recipe/meta.yaml

```
@@ -42,6 +42,7 @@ requirements:
 test:
   commands:
     - test -f $PREFIX/lib/libopenslide${SHLIB_EXT}  # [unix]
+    - env OPENSLIDE_DEBUG=synthetic slidetool prop list ""  # [unix]
```

```suggestion
    - slidetool --help
```
I think we just made a mistake in this repository, i don't think there is a particular need to add this test or a new build.

If you want to add a test, the help menu is better.

The test as proposed earlier exits with value 1, even if libdicom is installed.

---

i would like to add a test of `slidetool` to ensure the command line program works. we can catch runtime linking errors during conda builds with this.

---

The `OPENSLIDE_DEBUG=synthetic` test should not exit 1 unless there is a problem.  What's the benefit of skipping an inexpensive test that could catch bugs?

---

sorry i think i missed the environment variable. yo uare right, there no problem with the test, but i would keep it for "future work" and not release a package for this given we patching the old package right now to add this dependency.

---

just FYI it will take a while for the new version of libdicom to be made available.